pkg/cluster-operator/customcluster_controller_test.go

```diff
@@ -0,0 +1,100 @@
+package clusteroperator
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"kurator.dev/kurator/pkg/apis/infra/v1alpha1"
+	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	fake "sigs.k8s.io/controller-runtime/pkg/client/fake"
+)
+
+func TestDeleteWorkerPods(t *testing.T) {
+	//create a fake client
```

Comment:
nit：add a space after the double slashes (//) for better readability
replace `//create a fake client` with `// create a fake client`
and check the others